example/example.cpp

```diff
@@ -198,13 +201,28 @@ void trace_example()
 // stopwatch example
 #include "spdlog/stopwatch.h"
 #include <thread>
+#include "spdlog/sinks/udp_sink.h"
 void stopwatch_example()
 {
     spdlog::stopwatch sw;
     std::this_thread::sleep_for(std::chrono::milliseconds(123));
     spdlog::info("Stopwatch: {} seconds", sw);
 }
 
+void udp_example()
+{
+    spdlog::sinks::udp_sink_config cfg("127.0.0.1", 11091);
+    auto my_logger = spdlog::udp_logger_mt("udplog", cfg);
+    my_logger->set_level(spdlog::level::debug);
+    for (int i = 0; i < 10; i++) {
+        my_logger->info("hello world {}", i);
+#ifdef _WIN32
+        // sendto() on winsock will drop packets if sent too quickly
```

Comment:
It depends on lots of variables. Since this is just a samole I would keep it simple and remove the loop and the ifdef.